client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -102,7 +121,8 @@ export default function SitesTableRow( { site }: SiteTableRowProps ) {
 							rel="noreferrer"
 							title={ __( 'Visit Site' ) }
 						>
-							{ displaySiteUrl( site.URL ) }
+							<Icon size={ 16 } icon={ icon }></Icon>
```

Comment:
Any special reason not to self-close the tag?

```suggestion
							<Icon size={ 16 } icon={ icon } />
```

Reply:
> Any special reason not to self-close the tag?

Nope, nothing special.